hyperspy/model.py

```diff
@@ -29,6 +29,9 @@
 from scipy.optimize import (leastsq, least_squares,
                             minimize, differential_evolution)
 from scipy.linalg import svd
+from numpy.linalg import lstsq as linear
+from scipy.optimize import nnls
```

Comment:
this could go in line 29 as  `from scipy.optimize import (leastsq, least_squares, minimize, differential_evolution, nnls)`